### DISCUSSION TITLE
Two Dataset for Different Task

### DISCUSSION THREAD

**Discussion by MagicHealer** [ContinualAI/avalanche#1152] [Q&A] [answered]
Hi, May I ask if it is possible to use two datasets for different tasks. For example, we want to use ImageNet Dataset as the First task, and CUB200 as Secon Task. Is it Possible?

**Comment by AntonioCarta** [comment, marked as answer]
you can use [dataset_benchmark](https://avalanche-api.continualai.org/en/v0.2.1/generated/avalanche.benchmarks.generators.dataset_benchmark.html#avalanche.benchmarks.generators.dataset_benchmark).
